board/spi_flasher.h

```diff
@@ -53,6 +62,12 @@ void spi_flasher() {
     lastval = val;
 
     if (rcv && (DMA2->LISR & DMA_LISR_TCIF2)) {
+	  // handshake with esp
+	  // Set boot0 to output and pull high
+	  GPIOB->MODER &= ~(GPIO_MODER_MODER0);
+	  GPIOB->MODER |= GPIO_MODER_MODER0_0;
+	  GPIOB->ODR |= GPIO_ODR_OD0; 
```

Comment:
Can't figure why this is not building, I will try building and may be run it on an ESP I have.